cluster-autoscaler/FAQ.md

```diff
@@ -331,6 +332,23 @@ CA might scale down non-empty nodes with utilization below a threshold
 
 To prevent this behavior, set the utilization threshold to `0`.
 
+### How can I prevent Cluster Autoscaler from scaling up too quickly?
+
+Running many shortlived scheduled jobs in a regular intervals could lead to cluster running in
+cycle of scaling up, executing jobs quickly and then waiting for `scale-down-unneeded-time`
```

Comment:
Increasing `--scale-down-unneeded-time` is also a way of improving the situation. In a way the two flags are complementary: `--scale-down-unneeded-time` controls how long a node will wait for pods, `--new-pod-scale-up-delay` controls how long a pod will wait for a node.

Reply:
How would increasing `--scale-down-unneeded-time` improve the situation? You will just get nodes that are started running even longer unneeded?

Reply:
The way I understand the problem here is that there is too high node churn - too many nodes coming down and up again, wasting compute resources on startup/shutdown. Keeping nodes for a longer time after they become empty is a way of addressing this, so is introducing a delay before triggering a scale up.

If the problem is phrased as "scale up happens too quickly" then it can also help: nodes are not deleted, giving future pods a chance to get scheduled on them instead of triggering a scale up.

Reply:
My problem was that nodes are coming up too fast and then staying up too long. :smile: 
We did decrease `--scale-down-unneeded-time` to 5 minutes and it did help, but anyway, I understand your point. I will write in a way to cover different cases, depending on what is that you want to achieve.